ocs_ci/ocs/ocp.py

```diff
@@ -182,6 +185,9 @@ def get(
             dict: Dictionary represents a returned yaml file
         """
         resource_name = resource_name if resource_name else self.resource_name
+        selector = selector if selector else self.selector
+        if selector:
+            resource_name = ""
```

Comment:
I wonder if it breaks backward compatibility.
If we are 'get'ing with name AND selector, it should work too right?

Reply:
Actually this is behavior of OC client:
```console
$ oc get packagemanifest -n openshift-marketplace --selector "ocs-operator-dev=true" ocs-operator
error: name cannot be provided when a selector is specified
```
You are not able to use selector and the name of the resource at the same time.   

I didn't come out with better solution here.  We basically prefer selector over the name here.

I see the deployment passed here:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3827/console

I am running also whole tier1 to see if I didn't break anything.

Reply:
Ah ok.. Didn't know that (:
LGTM